docs/integrations/aws-org/discovery.md

```diff
@@ -0,0 +1,100 @@
+---
+id: discovery
+title: AWS Organization Discovery
```

Comment:
```suggestion
title: AWS Account Discovery
```

Does that make more sense? But that also makes me think should the rest change as well as in should it be "AWS Account Provider"? I'm not tied to this as someone who isn't actively using AWS I will defer but reading the docs here I came to this conclusion.

Reply:
I'm also mildly thinking about Adopters who see this as Org data in the context of Users and Groups. AWS has many services and I would not be surprised that they have something for Users and Groups

Reply:
@awanlin I just updated this to `AWS Organization Account Discovery`. I definitely could see the confusion since AWS has users/groups as well as AWS Identity Center 🤦 . Having organization in the name I think signals that the provider only works with folks using AWS Organizations to manage multiple accounts versus multiple standalone accounts (which isn’t best practice anyways). I’m open to the naming though, this was a tough one naming everything.

Reply:
Organizations have a pretty specific meaning in the context of AWS

https://docs.aws.amazon.com/organizations/latest/userguide/orgs_introduction.html

That being said we are really discovering accounts here not organizations, there organizations have to be explicitly listed.

Reply:
I went ahead and updated it to AWS Account discovery.